### DISCUSSION TITLE
I really like this plugin but it doesn't work properly in 1.21, please update it

### DISCUSSION THREAD

**Discussion by ximuya2** [dmulloy2/ProtocolLib#2444] [General]
I really like this plugin but it doesn't work properly in 1.21, please update it

**Comment by lukalt** [comment]
Hi, I suppose you mean Minecraft 1.20. You can download ProtocolLib for 1.20 here: https://ci.dmulloy2.net/job/ProtocolLib/654/.

Please keep in mind that all plugins that use ProtocolLib probably also need to be updated.
